docs/ug/withNetlify.md

```diff
@@ -10,9 +10,12 @@
 
 <div id="section-setting-up">
 
+<box type="important">
+Note that Netlify has a low limit for free tier users (only 300 _build minutes_ per month as at June 2020 -- a single report generation can take 2-3 build minutes, longer if your report includes many/big repositories). Due to this, we will not be supporting this method going forward.
```

Comment:
```suggestion
<box type="important">

Note that Netlify has a low limit for free tier users (only 300 _build minutes_ per month as at June 2020 -- a single report generation can take 2-3 build minutes, longer if your report includes many/big repositories). Due to this, we will not be supporting this method going forward.
```

There should be a blank line above a block of markdown text, for the markdown parser to process it as markdown text.